pkg/services/rpcsrv/server.go

```diff
@@ -2450,21 +2446,35 @@ func getRelayResult(err error, hash util.Uint256) (any, *neorpc.Error) {
 		return result.RelayResult{
 			Hash: hash,
 		}, nil
-	case errors.Is(err, core.ErrAlreadyExists):
+	case errors.Is(err, core.ErrTxExpired):
+		return nil, neorpc.WrapErrorWithData(neorpc.ErrExpiredTransaction, err.Error())
+	case errors.Is(err, core.ErrAlreadyExists) || errors.Is(err, core.ErrInvalidBlockIndex):
 		return nil, neorpc.WrapErrorWithData(neorpc.ErrAlreadyExists, err.Error())
+	case errors.Is(err, core.ErrAlreadyExistsMemPool):
```

Comment:
s/`core.ErrAlreadyExistsMemPool`/`core.ErrAlreadyInPool`

The relevant change should be present in this commit (the second commit).

Reply:
Fixed